src/main.py

```diff
@@ -85,30 +104,68 @@ def log_speed(
     # Return the current time and reset the counter to zero
     return time.time(), 0
 
-async def insert_data(batch: list[dict], error_queue:Queue):
-    try:
-        highscores:list[dict] = [msg.get("hiscores") for msg in batch]
-        players:list[dict] = [msg.get("player") for msg in batch]

```

Comment:
this one is key maybe its best we make the msg in batch a pydantic class with the keys hiscores & player